src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/controller/TierCalculationController.kt

```diff
@@ -28,9 +29,16 @@ class TierCalculationController(private val tierCalculationService: TierCalculat
       ApiResponse(code = 404, message = "Result Not Found")
     ]
   )
+
   @GetMapping("crn/{crn}/tier")
-  fun getNotifications(@PathVariable(required = true) crn: String): ResponseEntity<TierDto> {
-    return ResponseEntity.ok(tierCalculationService.getTierByCrn(crn))
+  fun getLatestTierCalculation(@PathVariable(required = true) crn: String): ResponseEntity<TierDto> {
```

Comment:
Do we need this endpoint now?

Reply:
I think we need to clarify this. I can't think of a specific use case, but we master the data now so need to make it available to other clients?

Reply:
Feels like we are doing half a job here. If there is a need for it we need to get the access restrictions and documentation etc right, and test it.